### PR TITLE
fix: remove nft-dev network

### PR DESCRIPTION
**Pull request by pdp2121**
## High Level Overview of Change

Title said it all.

### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### CODE REVIEW

**Review by mvadari — APPROVED** [review]

**Review by ckniffen — APPROVED** [review]

File: src/shared/database/setup.ts [hunk 1]
```diff
@@ -185,4 +185,10 @@ async function setupNetworksTable(): Promise<void> {
         .catch((err: Error) => log.error(err.message))
     }
   })
+  if (networksIds.includes('nft-dev')) {
```
Comment (ckniffen) [inline src/shared/database/setup.ts:188]: Will this cause foreign key errors?

Comment (pdp2121) [reply src/shared/database/setup.ts:188]: It doesnt use a foreign key. It just check whether nft-dev is in the array of ids of current networks on the table and if so delete that row using knex.

**Comment by ckniffen** [comment]
You should combine with #78

File: src/shared/database/setup.ts (hunk 1 above)
Comment (mvadari) [reply src/shared/database/setup.ts:192]: I don't think the networks table is used as a foreign key anywhere

**Review by mvadari — APPROVED** [review]

**Pull request merged**